week5

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+#define max(x, y) (x > y ? x : y)
+
+int main(void) {
+    int dice1 = 0, dice2 = 0, dice3 = 0, prize = 0;
+    scanf("%d %d %d", &dice1, &dice2, &dice3);
+
```

Comment:
Wow Amazing! You are very nice developer!

Reply:
Thanks!!